### PR TITLE
Fix update not working for Safes > 1.1.1

### PR DESCRIPTION
**Pull request by Uxio0**
- Update master copy function was removed after 1.1.1
- Migration contracts to update from 1.3.0 to 1.4.1 are still not available
- Test update

### CODE REVIEW

File: tests/test_safe_operator.py [hunk 1]
```diff
@@ -246,17 +263,42 @@ def test_send_ether(self):
...
+        with mock.patch.object(
+            MultiSend,
+            "MULTISEND_CALL_ONLY_ADDRESSES",
+            [self.multi_send_contract.address],
+        ):
+            safe_operator_v111.update_version()
+
+        self.assertEqual(
+            safe_operator_v111.safe.retrieve_master_copy_address(),
+            last_safe_contract_address_mock.return_value,
```
Comment (moisses89) [inline tests/test_safe_operator.py:296]:
```suggestion
            last_safe_contract_address_mock.return_value
```

File: tests/test_safe_operator.py [hunk 2]
```diff
@@ -246,17 +263,42 @@ def test_send_ether(self):
...
+        )
+        self.assertEqual(
+            safe_operator_v111.safe.retrieve_fallback_handler(),
+            last_default_fallback_handler_address.return_value,
```
Comment (moisses89) [inline tests/test_safe_operator.py:300]:
```suggestion
            last_default_fallback_handler_address.return_value
```

File: tests/test_safe_operator.py (hunk 1 above)
Comment (moisses89) [inline tests/test_safe_operator.py:296]: last_safe_contract_address_mock.return_value is safe_contract_V1_4_1.address, then the v1.1.1 was updated to 1.4.1 this is possible?

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: tests/test_safe_operator.py (hunk 1 above)
Comment (Uxio0) [reply tests/test_safe_operator.py]: Yes

**Review by moisses89 — APPROVED** [review]

**Pull request merged**